### PR TITLE
add new forces DLCs

### PR DESCRIPTION
**Pull request by NiklasMerz**
I still need to get the new pictures and the checkboxes are not ordered alphabetically, yet.

### CODE REVIEW

**Comment by NiklasMerz** [comment]
@MorpheusXAUT I added the new DLC logos to the static folder. Is the production backend updated so I can test against that? Running the backend locally seems a bit tricky because of the steam login. Or is there a way around to get logged in?

**Are you ok with the new DLCs at the end of the list or should we resort the table?**

**Comment by MorpheusXAUT** [comment]
> @MorpheusXAUT I added the new DLC logos to the static folder. Is the production backend updated so I can test against that? Running the backend locally seems a bit tricky because of the steam login. Or is there a way around to get logged in?
> 
> **Are you ok with the new DLCs at the end of the list or should we resort the table?**

Thanks, I've uploaded them to the production storage as well.
Just updated the production backend (v1.2.0), you should be able to test against it. There's no way around the Steam login, I'm afraid, but in _theory_ you can just generate your own Steam Web API Key for localhost [here](https://steamcommunity.com/dev/apikey) in case you want to test more complicated changes in the future.

I'm fine with the new DLCs being at the end since they're the latest - whatever looks best to you.

Thanks again for the willingness to help!

**Comment by NiklasMerz** [comment]
Thanks for updating the production backend. With copying the auth token I can use it for testing the frontend locally.

**I did a few tests and fixed a few more things and now it's looking good.** Please do another review as I'm pretty sure I might have forgotten something. Looking forward to get this live :rocket:

File: src/components/missions/modals/MissionEditModal.vue [hunk 1]
```diff
@@ -588,6 +596,8 @@ export default {
           return
         }
 
+        console.debug("update", dlc)
+
```
Comment (MorpheusXAUT) [inline src/components/missions/modals/MissionEditModal.vue]:
```suggestion
```

Could you remove the debug log, just to clean up a bit?

File: src/components/missions/modals/MissionSlotEditModal.vue
Comment (MorpheusXAUT) [inline src/components/missions/modals/MissionSlotEditModal.vue:1]:
```
      this.missionSlotEditRequiredDLCs = {
        aow: this.hasSlotRequiredDLC('aow'),
        apex: this.hasSlotRequiredDLC('apex'),
        contact: this.hasSlotRequiredDLC('contact'),
        csla: this.hasSlotRequiredDLC('csla'),
        ef: this.hasSlotRequiredDLC('ef'),
        gm: this.hasSlotRequiredDLC('gm'),
        helicopters: this.hasSlotRequiredDLC('helicopters'),
        jets: this.hasSlotRequiredDLC('jets'),
        karts: this.hasSlotRequiredDLC('karts'),
        lawsofwar: this.hasSlotRequiredDLC('laws-of-war'),
        marksmen: this.hasSlotRequiredDLC('marksmen'),
        rf: this.hasSlotRequiredDLC('rf'),
        tacops: this.hasSlotRequiredDLC('tac-ops'),
        tanks: this.hasSlotRequiredDLC('tanks'),
        vn: this.hasSlotRequiredDLC('vn')
      }
```

We're missing the new DLCs around L294

**Review by MorpheusXAUT — CHANGES_REQUESTED** [review]
With these two minor changes, LGTM! Thanks again for the work, I'll get it merged and deployed as soon as these are resolved.

**Review by MorpheusXAUT — APPROVED** [review]

**Comment by NiklasMerz** [comment]
Thanks for the review. I missed those!

**Comment by MorpheusXAUT** [comment]
Deployed and doesn't seem to be crashing instantly on page load 😋

**Pull request merged**